utils/s2n_safety.c

```diff
@@ -148,3 +150,17 @@ int s2n_constant_time_pkcs1_unpad_or_dont(uint8_t * dst, const uint8_t * src, ui
 
     return 0;
 }
+
+static bool s_s2n_in_unit_test = false;
+
+bool s2n_in_unit_test()
+{
+    return s_s2n_in_unit_test;
+}
+
+bool s2n_in_unit_test_set(bool newval)
```

Comment:
Should this return `0`, or the previous value? Which makes more sense in the s2n api?